message/validation/pubsub_validation.go

```diff
@@ -10,8 +10,14 @@ func (mv *messageValidator) validatePubSubMessage(pMsg *pubsub.Message) error {
 		return ErrPubSubMessageHasNoData
 	}
 
+	maxMsgSize := MaxEncodedMsgSize
+
+	if mv.netCfg.Beacon.EstimatedCurrentEpoch() >= mv.pectraForkEpoch {
```

Comment:
This would look better if it was a method on the network config, but as I said in the comment in `cli/operator/node.go`, we'll probably need to do it in/after https://github.com/ssvlabs/ssv/pull/1308. I'd leave a TODO though

Reply:
to be honest I don't think this comment here is needed because it will be inevitable to change it once we apply #1308 or similar change.

Reply:
https://github.com/ssvlabs/ssv/pull/2088#discussion_r1998836873